src/components/AddActivityForm.test.js

```diff
@@ -3,34 +3,56 @@ import '@testing-library/jest-dom'
 import '@testing-library/jest-dom/extend-expect'
 import { render, screen, fireEvent } from '@testing-library/react'
 import AddActivityForm from './AddActivityForm'
+import { createMemoryHistory } from 'history'
+import { Router } from 'react-router'
 
 describe('AddActivityForm.test.js', () => {
   it('should display the entered value of the name input field after change event', () => {
-    render(<AddActivityForm />)
+    const history = createMemoryHistory()
```

Comment:
die Zeile hier kann man vielleicht in ein beforeEach packen, das hast du glaube ich überall

Reply:
Danke, werde ich auch machen!